download.dd

```diff
@@ -1,9 +1,5 @@
 Ddoc
 
-<!-- Github ribbon -->
-<a href="http://github.com/D-Programming-Language"><img id="github-ribbon"
-src="images/github-ribbon.png" alt="Fork D on GitHub"></a>
-
```

Comment:
Since the GitHub ribbon is removed, I think there should be a link to the source code somewhere (e.g. near the release archive / GPG keys links).